### DISCUSSION TITLE
Pressure setting trouble

### DISCUSSION THREAD

**Discussion by ilkerbesken** [LinwoodDev/Butterfly#229] [Q&A] [answered]
Hi there,

I am very glad that it is an alternative to onenote and similar note-taking applications. Butterfly is really useful and has many good abilities. Pressure sensitivity is one of them. But I can only feel the pressure sensitivity when using it on the web. I tried with android, windows and linux but I couldn't feel the pressure sensitivity. ( I'm using it with my Wacom intuos 490 graphics tablet. )

Is this just a problem I'm having or is it something that's planned to be fixed for now?

Thanks to the beautiful people who developed this application for us.

**Comment by CodeDoctorDE** [comment, marked as answer]
Android should work without problems.
I'm using the input provided by the flutter framework.
There are already issues to track it:
Windows: https://github.com/flutter/flutter/issues/65248 and https://github.com/flutter/flutter/issues/102836
Linux: https://github.com/flutter/flutter/issues/63209

These issues are a bit older. The web version should work fine for now.

**Reply by ilkerbesken** [reply]
Thanks for your quick reply. 

I didn't search previous report of issues, it's my fault. I will follow them.

But touch sensitivity doesnt work with my mobile. (It is samsung galaxy s20 with android 12)
I dont have any stylus to write on my mobile. Maybe it can detect sensitivity with the stylus. I don't know as I haven't tried this. But touch sensitivity doesnt work for me.

Web version is best option right now.

Thanks a lot!

**Reply by CodeDoctorDE** [reply]
Oh no problem. I'm also interested to see this.
There isn't something like a touch sensitivity. For getting the pressure you need an extra hardware (a stylus pen and something on your device) to use it. Come to the discord/matrix server if you have additional questions.
